_pages/en_US/include/install-boot9strap-safeb9sinstaller.txt

```diff
@@ -3,6 +3,7 @@ In this section, you will install custom firmware onto your device.
 {%- endif %}
 
 1. When prompted, input the key combo given on the top screen to install boot9strap
+  + If a step on the lower screen becomes red, [follow this troubleshooting guide](troublshooting#issues-with-safeb9sinstaller)
```

Comment:
a step is unlikely to 'become red', one of the options will likely show up as red without the user doing anything (since stuff that could cause issues later, like otp crypto fail and firm backup fail, are not currently addressed by troubleshooting)